usr/share/rear/build/OPALPBA/Linux-i386/391_list_executable_dependencies.sh

```diff
@@ -0,0 +1,16 @@
+# List executables along with the shared libraries they depend on
+#
+# The resulting list can be used to determine which executables with a large footprint could be stripped from the
+# TCG Opal pre-boot authentication (PBA) image
+
+if is_true $KEEP_BUILD_DIR; then
```

Comment:
`KEEP_BUILD_DIR` is not ternary, it is boolean, i.e. empty means false, nonempty means true.

Reply:
In #2218 we are discussing the possibility of making it ternary though.